schema/collation/test_schema.json

```diff
@@ -111,7 +111,11 @@
           "warning": {
             "description": "Present when data has a problem",
             "type": "string"
-          }
+          },
+          "hash_id": {
+            "description": "Has value of the JSON string omitting label value",
+            "type": "integer"
+         }
```

Comment:
Question: I don't see `hash_id` used anywhere in this PR outside the schema. Why do you need it?